src/theme.js

```diff
@@ -65,6 +65,12 @@ const theme = {
     default: "#007fbf",
     light: "#a3c7d9",
   },
+  input: {
+    border: "0.25em solid #a3c7d9",
+    padding: "0.5em",
+    autofillBgColor: "#ffffe0",
+    defaultBgColor: "#ffffff",
+  },
```

Comment:
Yeah that's a good point - we don't have any criteria for our theme yet. It seems like the theme should include any styles that would be overridden by the flavor. So in theory, this could be all styles, but we have to model this somehow. So I think allowing a flavor to override the style of an input seems YAGNI.